docs/Lama2/docs/explanation/l2format.md

```diff
@@ -83,6 +83,17 @@ Cookies are specified in a `Cookie` header as follows:
 Cookie:'sessionid=foo;another-cookie=bar'
 ```
 
+### Global Environments variables/commands can be defined in `/l2config.env/<requests_dir>`
+
+The *l2* loads up variables from `l2config.env` and then `l2.env`.
```

Comment:
Again - the overwrite explanation is more complex to the reader/user. Simply say what order in which variables are searched for (nearest first):

1. VM
2. local
3. global

Reply:
Either:

`The l2 binary loads...`

or

`l2` loads